osd/cluster-wide_proxy_network_misconfiguration.json

```diff
@@ -0,0 +1,7 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "summary": "Action required: Cluster-wide proxy Network misconfigration",
+    "description": "Your cluster requires you to take action. SRE has observed that there have been changes made to network configuration which impacts normal working of the cluster with proxy, including lack of network egress to Internet-based resources which are required for cluster operation and support.",
```

Comment:
Can you clarify which situation(s) this SL would be sent for that the `cluster-wide_proxy_unreachable` one is not suitable for?

Reply:
I was thinking this in perspective of something change in routing table or in security group specifically which makes proxy not reachable from openshift cluster. But it seems `cluster-wide_proxy_unreachable.json` covers the same with ${REASON} = network configuration changes. Thanks